src/proc/rotation-filter.cpp

```diff
@@ -95,24 +95,21 @@ namespace librealsense {
 
         if (auto tgt = prepare_target_frame(f, source, tgt_type))
         {
-            int rotated_width = ( _value == 90 || _value == -90 ) ? src.get_height() : src.get_width();
-            int rotated_height = ( _value == 90 || _value == -90 ) ? src.get_width() : src.get_height();
-
             switch( bpp )
             {
             case 1: {
-                rotate_depth< 1 >( static_cast< uint8_t * >( const_cast< void * >( tgt.get_data() ) ),
+                rotate_frame< 1 >( static_cast< uint8_t * >( const_cast< void * >( tgt.get_data() ) ),
                                    static_cast< const uint8_t * >( src.get_data() ),
-                                   rotated_height,
-                                   rotated_width );
+                                   src.get_height(),
+                                   src.get_width() );
                 break;
             }
 
             case 2: {
-                rotate_depth< 2 >( static_cast< uint8_t * >( const_cast< void * >( tgt.get_data() ) ),
-                                   static_cast< const uint8_t * >( src.get_data() ),
-                                   rotated_height,
-                                   rotated_width );
+                rotate_frame< 2 >( static_cast< uint8_t * >( const_cast< void * >( tgt.get_data() ) ),
+                                   static_cast< const uint8_t * >( src.get_data() ) ,
+                                   src.get_height(),
```

Comment:
Normally the convention is width, height 
![image](https://github.com/user-attachments/assets/75738edf-732d-40fe-af7d-a248cc2aedce)